operations/iuf/workflows/backup.md

```diff
@@ -3,14 +3,14 @@
 This section describes procedures that back up critical state in case it becomes necessary to fall back to previous configurations and software.
 
 - [1. Backup](#1-backup)
-  - [1.1 Slingshot Fabric Manager](#11-slingshot-fabric-manager)
+    - [1.1 Slingshot Fabric Manager](#11-slingshot-fabric-manager)
 - [2. Next steps](#2-next-steps)
 
 ## 1. Backup
 
 ### 1.1 Slingshot Fabric Manager
 
-Create a backup of the Slingshot Fabric Manager prior to proceeding with the workflow. Refer to the "Backup and Restore Operation of Fabric Configuration" section in the _Slingshot Operations Guide for Customers_
+Create a backup of the Slingshot Fabric Manager prior to proceeding with the workflow. Refer to the "Backup of a Fabric Manager configuration" section in the _Slingshot Operations Guide for Customers_
```

Comment:
```suggestion
Create a backup of the HPE Slingshot Fabric Manager prior to proceeding with the workflow. Refer to the "Backup of a Fabric Manager configuration" section in the _HPE Slingshot Operations Guide_
```
Note that starting in Slingshot 2.2, the document is no longer titled the _HPE Slingshot Operations Guide_. The document to reference will be the _HPE Slingshot Installation Guide for CSM_.

Reply:
Addressed this review comment